### PR TITLE
Add custom action to propagate env change

### LINKED ISSUE #2560: Windows PI install should also add path to C:\Program Files (x86)\Microsoft SDKs\Azure\CLI\bin
Windows install suggestion from internal customer @timlt who uses Windows Git Bash shell: Windows CLI install should set path C:\Program Files (x86)\Microsoft SDKs\Azure\CLI\**bin** in addition to C:\Program Files (x86)\Microsoft SDKs\Azure\CLI\wbin . This will enable the CLI to work system-wide for shells other than CMD or PSH. 

His experience trying to run Azure CLI in GitBash shell was that the commands weren't found until he manually added C:\Program Files (x86)\Microsoft SDKs\Azure\CLI\bin to his PATH environment variable.

### PR DESCRIPTION
**Pull request by yugangw-msft**
jacques, do i need to sign the exe?
Fix #2560 and #2388

### CODE REVIEW

**Comment by joeloff** [comment]
@yugangw-msft yes, if you're dropping the EXE on disk then it needs to be signed. Make sure you authenticodesign it with the SHA2 keys as SHA1 is no longer considered valid

File: tools/windows/Product.wxs [hunk 1]
```diff
@@ -39,6 +39,19 @@
...
 		</Feature>
 
+    <!--Custom action to propagate path env variable change-->
+    <Binary Id="PropagateEnvChangeExe" SourceFile=".\propagate_env_change\propagate_env_change.exe" />
+    <Property Id="WixQuietExecCmdLine" Value="propagate_env_change.exe"/>
+    <!--Ignoring the return value if case there is app-hang we can't control -->
+    <CustomAction Id="PropagateEnvChange" 
+                  BinaryKey="PropagateEnvChangeExe" 
+                  ExeCommand="propagate_env_change.exe" 
+                  Execute="commit"
```
Comment (joeloff) [inline tools/windows/Product.wxs]: Just FYI Setting execute='commit' will not execute the action during a rollback if the installation fails

Comment (yugangw-msft) [inline tools/windows/Product.wxs]: this is fine

**Comment by yugangw-msft** [comment]
@amarzavery we will need to test out at least on  win7 and 8 machines before merge

**Comment by yugangw-msft** [comment]
@joeloff , thanks, I have signed the exe.

**Pull request merged**